services/libfuzzer/README.md

```diff
@@ -1,17 +1,41 @@
 ## Local Development
 
-The production Dockerfile uses credstash to fetch credentials for our private FuzzManager instance. If you do not have the credentials of your FuzzManager instance stored in a KMS or similar database and/or you want to develop on this image then the following approach is recommended.
+The production Dockerfile uses Taskcluster secrets to fetch credentials for our private FuzzManager instance. If you do not have the credentials of your FuzzManager instance stored in a KMS or similar database and/or you want to develop on this image then the following examples are recommended.
+
+Building locally:
+
+    TAG=dev
+    docker build -t mozillasecurity/libfuzzer:$TAG ../.. -f Dockerfile
+
+... or to test the latest build:
+
+    TAG=latest
+
+Testing locally:
+
+    eval $(TASKCLUSTER_ROOT_URL=https://community-tc.services.mozilla.com taskcluster signin)
+    LOGS="logs-$(date +%Y%m%d%H%M%S)"
+    mkdir -p "$LOGS"
+    docker run --rm -it \
+        -e TASKCLUSTER_ROOT_URL \
+        -e TASKCLUSTER_CLIENT_ID \
+        -e TASKCLUSTER_ACCESS_TOKEN \
+        -v "$(pwd)/$LOGS":/logs \
+        mozillasecurity/libfuzzer:$TAG 2>&1 | tee "$LOGS/live.log"
+
+... add any environment variables required by the fuzzer using `-e VAR=value`
 
 ### Example: LibFuzzer Coverage Run
 
 ```bash
-REVISION=$(curl -sL https://build.fuzzing.mozilla.org/builds/coverage-revision.txt)
-fuzzfetch --build "$REVISION" --fuzzing --coverage -a --gtest -n firefox
+REVISION="$(curl --compressed -sSL https://community-tc.services.mozilla.com/api/index/v1/task/project.fuzzing.coverage-revision.latest/artifacts/public/coverage-revision.txt)"
+fuzzfetch --build "$REVISION" --fuzzing --coverage --gtest -n firefox
```

Comment:
The gtest artifact does not exists for this build: https://firefox-ci-tc.services.mozilla.com/tasks/index/gecko.v2.mozilla-central.latest.firefox/linux64-ccov-fuzzing-opt